sdk/src/test/java/co/omise/android/ConfigTest.java

```diff
@@ -0,0 +1,39 @@
+package co.omise.android;
+
+import android.os.Build;
+
+import org.junit.Test;
+
+import co.omise.android.api.Config;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfigTest {
+    private static final String MODEL = Build.MODEL;
+    private static final String PKG_VERSION = BuildConfig.VERSION_NAME;
+    static final String API_VERSION = "new-shiny-version";
+    private static final int ANDROID_VERSION = Build.VERSION.SDK_INT;
+    static final String PKEY = "pkey_test_123";
+
+    @Test
+    public void testConfig() {
```

Comment:
I think the test naming not quite right. Could follow the convention name?

Reply:
Done 👍